application/gqlgen/mutations.go

```diff
@@ -162,3 +162,51 @@ func (r *mutationResolver) SetThreadLastViewedAt(ctx context.Context, input SetT
 
 	return &thread, nil
 }
+
+// CreateTrust establishes a Trust between two organizations
+func (r *mutationResolver) CreateTrust(ctx context.Context, input CreateTrustInput) (*models.Organization, error) {
+	cUser := models.GetCurrentUserFromGqlContext(ctx)
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+	}
+
+	var organization models.Organization
+	if err := organization.FindByUUID(input.PrimaryID); err != nil {
+		return nil, reportError(ctx, err, "CreateTrust.FindPrimaryOrganization", extras)
+	}
+
+	if !cUser.CanEditOrganization(organization.ID) {
```

Comment:
Maybe use a new CanCreateTrust function that is more strict.

Reply:
Sounds good.